packages/react/src/utils/positioner/Popper.tsx

```diff
@@ -217,7 +217,9 @@ const Popper: React.FunctionComponent<PopperProps> = props => {
       ? children({ placement: computedPlacement, scheduleUpdate })
       : (children as React.ReactElement)
 
-  return <Ref innerRef={contentRef}>{React.Children.only(child)}</Ref>
+  return child ? (
+    <Ref innerRef={contentRef}>{React.Children.only(child) as React.ReactElement}</Ref>
+  ) : null
```

Comment:
Not sure that it's enough safe. I'd rather try to avoid rendering of `Popper` component at all.

Reply:
Oops, we have already done this change. Ignore